generators/client/templates/react/src/main/webapp/app/reducers/account.ts

```diff
@@ -5,7 +5,7 @@ import { messages } from '../config/constants';
 import { ICrudPutAction } from '../shared/model/redux-action.type';
 
 export const ACTION_TYPES = {
-  CREATE_PROPERTY: 'systemProperty/CREATE_PROJECT',
+  CREATE_PROPERTY: 'systemProperty/CREATE_PROPERTY',
```

Comment:
Ah missed it. Actually this reducer doesnt need CREATE_PROPERTY and also it should be `account` and not systemProperty. This needs some cleanup. Would you like to do it?